intake/auth/base.py

```diff
@@ -0,0 +1,18 @@
+
+
+class BasicAuth(object):
```

Comment:
I'd rename this to `BaseAuth` or `BaseAuthPlugin`.  `BasicAuth` is likely to be confused with HTTP Basic Authentication.